packages/snap-preact/components/src/components/Atoms/Icon/Icon.stories.tsx

```diff
@@ -72,15 +72,6 @@ export default {
 			},
 			control: { type: 'none' },
 		},
-		title: {
-			description: 'Title text to render inside the svg',
-			table: {
-				type: {
-					summary: 'string',
-				},
-			},
-			control: { type: 'text' },
-		},
```

Comment:
This is a valid prop documented in the readme, why remove it from Storybook?

Reply:
twas a duplicate